src/client/src/orangehrmRecruitmentPlugin/pages/ViewCandidate.vue

```diff
@@ -0,0 +1,436 @@
+
+        <oxd-form-actions>
+          <oxd-button
+            display-type="ghost"
+            :label="$t('general.reset')"
+            @click="onClickReset"
+          />
+          <oxd-button
+            class="orangehrm-left-space"
+            display-type="secondary"
+            :label="$t('general.search')"
+            type="submit"
+          />
+        </oxd-form-actions>
+      </oxd-form>
+    </oxd-table-filter>
+    <br />
+    <div class="orangehrm-paper-container">
+      <div class="orangehrm-header-container">
+        <oxd-button
+          :label="$t('general.add')"
+          icon-name="plus"
+          display-type="secondary"
+          @click="onClickAdd"
+        />
+      </div>
+      <table-header
+        :selected="checkedItems.length"
+        :total="total"
+        :loading="isLoading"
+        @delete="onClickDeleteSelected"
+      ></table-header>
+      <div class="orangehrm-container">
+        <oxd-card-table
+          v-model:selected="checkedItems"
+          v-model:order="sortDefinition"
+          :headers="headers"
+          :items="items?.data"
+          :selectable="true"
+          :clickable="false"
+          :loading="isLoading"
+          row-decorator="oxd-table-decorator-card"
+        />
+      </div>
+      <div class="orangehrm-bottom-container">
+        <oxd-pagination
+          v-if="showPaginator"
+          v-model:current="currentPage"
+          :length="pages"
+        />
+      </div>
+    </div>
+    <delete-confirmation ref="deleteDialog"></delete-confirmation>
+  </div>
+</template>
+
+<script>
+import {computed, ref} from 'vue';
+import useSort from '@ohrm/core/util/composable/useSort';
+import usePaginate from '@ohrm/core/util/composable/usePaginate';
+import {navigate} from '@ohrm/core/util/helper/navigation';
+import {APIService} from '@/core/util/services/api.service';
+import JobtitleDropdown from '@/orangehrmPimPlugin/components/JobtitleDropdown';
+import VacancyDropdown from '@/orangehrmRecruitmentPlugin/components/VacancyDropdown';
+import HiringManagerDropdown from '@/orangehrmRecruitmentPlugin/components/HiringManagerDropdown';
+import CandidateAutocomplete from '@/orangehrmRecruitmentPlugin/components/CandidateAutocomplete';
+import DeleteConfirmationDialog from '@ohrm/components/dialogs/DeleteConfirmationDialog';
+import usei18n from '@/core/util/composable/usei18n';
+const defaultFilters = {
+  jobTitle: null,
+  vacancy: null,
+  hiringManager: null,
+  status: null,
+  keywords: null,
+  application: null,
+  candidate: '',
+  dateFrom: '',
+  dateTo: '',
+};
+const defaultSortOrder = {
+  'v.vacancy': 'ASC',
+  'c.candidate': 'ASC',
+  'h.manager': 'ASC',
+  's.status': 'DEFAULT',
+};
+
+export default {
+  name: 'ViewCandidate',
+  components: {
+    CandidateAutocomplete,
+    HiringManagerDropdown,
+    VacancyDropdown,
+    JobtitleDropdown,
+    'delete-confirmation': DeleteConfirmationDialog,
+  },
+  props: {
+    unselectableIds: {
```

Comment:
according to spec are there undeletable candidates?

Reply:
i think no